radis/io/hdf5.py

```diff
@@ -213,6 +213,14 @@ def combine_temp_batch_files(self, file, key="default", sort_values=None):
             import vaex
 
             df = vaex.open(self._temp_batch_files, group=key)
+            # Removing Nan values columns
```

Comment:
minor suggestion, but maybe add a `delete_nan_columns`  parameter  (default True)  in combine_temp_batch_files() and run these lines only if is is True